From 417f44124e34815e98aa7e736978f5763cdb19a7 Mon Sep 17 00:00:00 2001
From: Paul Ganssle <paul@ganssle.io>
Date: Fri, 17 May 2019 18:50:09 -0400
Subject: [PATCH] Register pytest markers

The latest version of pytest has started warning on unregistered
markers, which the test suite uses fairly extensively, causing the tests
to fail. This explicitly registers all the markers used in the test
suite.

--- setup.cfg.orig	2019-02-05 14:11:22 UTC
+++ setup.cfg
@@ -14,6 +14,20 @@ filterwarnings = 
 	error
 	error::DeprecationWarning
 	error:PendingDeprecationWarning
+markers =
+    gettz
+    import_star
+    isoparser
+    parserinfo
+    rrule
+    rruleset
+    rrulestr
+    smoke
+    tz_resolve_imaginary
+    tzfile
+    tzlocal
+    tzoffset
+    tzstr
 
 [egg_info]
 tag_build = 
